src/BloomBrowserUI/bookEdit/js/bloomEditing.js

```diff
@@ -725,6 +725,18 @@ var pageSelectionChanging = function () {
     fireCSharpEditEvent('finishSavingPage', '');
 };
 
+// This is invoked from C# when we are about to leave a page (often right after the previous
+// method for changing pages).  It is mainly to clean things up so that garbage collection
+// won't lose multiple megabytes of data that both the DOM (C++) and Javascript subsystems
+// think the other is still using.
+var cleanupPageUponDeparture = function () {
+    var marginBox = $('.marginBox');
```

Comment:
I don't know that there's any reason for starting there.  I'd be happy to start it at a higher element (even body, i suppose).